student_explorer/settings.py

```diff
@@ -260,14 +260,14 @@
         BASE_DIR, 'student_explorer/local/saml/student-explorer-saml.pem'),
       }
 
-SAML_CREATE_UNKNOWN_USER = False
-
-SAML_ATTRIBUTE_MAPPING = {
-    'uid': ('username', ),
-    'mail': ('email', ),
-    'givenName': ('first_name', ),
-    'sn': ('last_name', ),
-}
+    SAML_CREATE_UNKNOWN_USER = True
```

Comment:
Comment here for why setting SAML_CREATE_UNKNOWN_USER to be true?

Reply:
This is based on the flowchart on #121 

- Is user in M-Community Group (yes) -> Does user have an SE account (no) -> SE account is created.